docs/en/cue/cross-namespace-resource.md

```diff
@@ -0,0 +1,54 @@
+# Define resources located in defferent namespace with application
+
+In this section, we will introduce how to use cue template create cross namespace resources (workload/trait)
+
+Generally you don't need to set metadata.namespace field in cue template. vela will create them in the same namespace with applicaiton.
+
+But in some cases maybe you want resources running in a specific namespace witch is different with application, you can set the metadata.namespace field.
+And vela will create the resources in the specified namespace, and create a resourceTracker object as owener of those resources
+
+
+## Usage
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  name: worker
+spec:
+  definitionRef:
+    name: deployments.apps
+  schematic:
+    cue:
+      template: |
+        parameter: {
+        	name:  string
+        	image: string
+        	namespace: string  # make this parameter `namespace` as keyword which represents the resource maybe located in defferent namespace with application
+        }
+        output: {
+        	apiVersion: "apps/v1"
+        	kind:       "Deployment"
+                metadata: {
+                   namespace: my-namespace
+                }
+        	spec: {
+        		selector: matchLabels: {
+        			"app.oam.dev/component": parameter.name
+        		}
+        		template: {
+        			metadata: labels: {
+        				"app.oam.dev/component": parameter.name
+        			}
+        			spec: {
+        				containers: [{
+        					name:  parameter.name
+        					image: parameter.image
+        				}]
+        			}}}
+        }
+```
+## Limitations
+If you update definition by changing the metadata.namespace field. vela will create new resources in the new namespace but not delete old resources.
+You need delete the application and recreate it to make the status of resource right
```

Comment:
```suggestion
We wil fix the limitation in the near future.
```

Reply:
We should never recommend our users do any un-reasoning things. And we have discussed that we should fix it ASAP.

Reply:
Got it